src/java/org/apache/cassandra/service/accord/AccordSerializers.java

```diff
@@ -54,23 +57,24 @@
 import static org.apache.cassandra.db.marshal.CollectionType.Kind.LIST;
 import static org.apache.cassandra.db.marshal.CollectionType.Kind.MAP;
 import static org.apache.cassandra.db.marshal.CollectionType.Kind.SET;
+import static org.apache.cassandra.utils.ByteBufferUtil.nullableByteBufferSerializer;
 
 public class AccordSerializers
 {
+    public static <T> EmbeddedAsymmetricVersionedSerializer<T, T, Version> embedded(VersionedSerializer<T, Version> serializer)
```

Comment:
TODO: remove these methods, they are kinda unsafe as we odn't know the best version to use at this layer